### ISSUE TITLE
Breaks await expression

### ISSUE THREAD

**Issue by salomvary** [assetgraph/assetgraph-builder#649]
Given an input file named `test.html` that looks like this:

```html
<script>
async function test () {
	const result = await (false ? Promise.resolve(null) : Promise.resolve('hello'))
	console.log(typeof result, result)
}

test()
</script>
```

Running assetgraph-builder with the following arguments:

```
buildProduction --nocompress --root . --outroot dist test.html
``` 

Results in the following `dist/test.html`:

```html
<script>async function test(){const result=await false?Promise.resolve(null):Promise.resolve('hello');;}test()</script>
```

Which is wrong. `await (foo ? bar : baz)` is [not the same as](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) `await foo ? bar : baz`!

Using `assetgraph-builder@6.4.2`.

**Comment by salomvary** [comment]
Potentially related: https://github.com/estools/escodegen/issues/384

**Comment by papandreou** [comment]
Thanks for reporting this. I think you're absolutely right that it's due to that escodegen issue. We should try to get that fixed ASAP!

**Comment by papandreou** [comment]
Fix submitted upstream here: https://github.com/estools/escodegen/pull/389

Switched to a fork with the fix and released 6.4.3.

**Comment by salomvary** [comment]
@papandreou that was fast, thanks!
